src/Codeception/Test/Feature/CodeCoverage.php

```diff
@@ -26,8 +26,20 @@ public function codeCoverageEnd($status, $time)
             return;
         }
 
+        if (method_exists($this, 'getLinesToBeCovered')) {
```

Comment:
That's correct. As far as I'm aware, there's nowhere to actually put the
annotations in tests of those formats.

An alternative implementation of the methods in this interface would still
allow a test writer to control code coverage, it would just have to be
configured differently by the test writer. Hooking ´$I->wantToTest()´ or a
similar "comment" method and parsing out what one might put in a `@covers`
might work for Cept, for example.

In addition to `@covers` and ´@uses´, this change also enables
`@codeCoverageIgnore` since it's controlled using the same mechanism.